Essentials/src/com/earth2me/essentials/commands/Commandback.java

```diff
@@ -13,24 +20,82 @@ public Commandback() {
     }
 
     @Override
-    protected void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+    protected void run(Server server, User user, String commandLabel, String[] args) throws Exception {
+        CommandSource sender = user.getSource();
+        if (args.length > 0 && user.isAuthorized("essentials.back.others")) {
```

Comment:
This permission should be passed as the second argument to the super constructor [(example)](https://github.com/EssentialsX/Essentials/blob/25ddaef423fec89d090b3448baea8080c7aba0dd/Essentials/src/com/earth2me/essentials/commands/Commandtpauto.java#L12).

Reply:
It's a regular `EssentialsCommand` so that constructor doesn't exist, unless I've missed something here...

Reply:
My mistake, forgot that was only there for toggle commands.